File: app/src/main/java/com/simplemobiletools/musicplayer/activities/LicenseActivity.java
package com.simplemobiletools.musicplayer.activities;

import android.content.Intent;
import android.net.Uri;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.simplemobiletools.musicplayer.R;

import butterknife.ButterKnife;
import butterknife.OnClick;

public class LicenseActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_license);
        ButterKnife.bind(this);
    }

    @OnClick(R.id.license_butterknife_title)
    public void butterKnifeClicked() {
        openUrl(R.string.butterknife_url);
    }

    @OnClick(R.id.license_ambilwarna_title)
    public void ambilwarnaClicked() {
        openUrl(R.string.ambilwarna_url);
    }

    @OnClick(R.id.license_otto_title)
    public void ottoClicked() {
        openUrl(R.string.otto_url);
    }

    private void openUrl(int id) {
        final String url = getResources().getString(id);
        final Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
        startActivity(browserIntent);
    }
}

File: app/src/main/kotlin/com/simplemobiletools/musicplayer/activities/LicenseActivity.kt
package com.simplemobiletools.musicplayer.activities

import android.content.Intent
import android.net.Uri
import android.os.Bundle
import com.simplemobiletools.musicplayer.R
import kotlinx.android.synthetic.main.activity_license.*

class LicenseActivity : SimpleActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_license)

        license_kotlin_title.setOnClickListener { openUrl(R.string.kotlin_url) }
        license_ambilwarna_title.setOnClickListener { openUrl(R.string.ambilwarna_url) }
        license_otto_title.setOnClickListener { openUrl(R.string.otto_url) }
        license_filepicker_title.setOnClickListener { openUrl(R.string.filepicker_url) }
        license_fileproperties_title.setOnClickListener { openUrl(R.string.fileproperties_url) }
    }

    private fun openUrl(id: Int) {
        val url = resources.getString(id)
        val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(url))
        startActivity(browserIntent)
    }
}
